resources/ext/build-scripts/install-vendored-gems.sh

```diff
@@ -19,5 +19,5 @@ while read LINE
 do
   gem_name=$(echo $LINE |awk '{print $1}')
   gem_version=$(echo $LINE |awk '{print $2}')
-  java -cp puppet-server-release.jar clojure.main -m puppetlabs.puppetserver.cli.gem --config jruby.conf -- install ${gem_name} --version ${gem_version}
+  java -cp puppet-server-release.jar:jruby-1_7.jar clojure.main -m puppetlabs.puppetserver.cli.gem --config jruby.conf -- install ${gem_name} --version ${gem_version}
```

Comment:
Yeah, all of the gem versions we have pinned in there currently should work fine with Ruby language version 1.9.3.  Hopefully that's the case as well for JRuby 9k (currently Ruby language version 2.3.1) - at least I haven't seen any aberrant behavior from them when running Puppet acceptance tests under JRuby 9k so far.

At some point in time, it does seem like we could run into a situation where we need to have different sets of gems for 1.7 vs 9k.  At that point, we may need to modify this script to bundle gems into separate directories for each and modify the Puppet Server source to setup the appropriate vendor gem path by default for the current JRuby version in use.  I was hoping to avoid having to look into that until we may need to do it.